### ISSUE TITLE
Missing parameters of layers

### ISSUE THREAD

**Issue by KevinzhouCUC** [Arthur151/ROMP#360]
Thanks so much for sharing the code and solving problems.
I've encounterd a similiar problem as #116 but not the same. I have already downloaded the fine tuned model ROMP_HRNet32_V1.pkl
but it still cannot process my images.

python -m romp.predict.image --inputs /root/autodl-tmp/data/my_video/output/images --output_dir /root/autodl-tmp/data/my_video/output/smpl_pred

No configs_yml is set, set it to the default --configs_yml=configs/image.yml
yaml_timestamp  /root/autodl-tmp/neuman/preprocess/ROMP/active_configs/active_context_2022-11-01_21_11_05.yaml
Loading the configurations from configs/image.yml
INFO:root:{'tab': 'hrnet_cm64_process_images', 'configs_yml': 'configs/image.yml', 'inputs': '/root/autodl-tmp/data/my_video/output/images', 'output_dir': '/root/autodl-tmp/data/my_video/output/smpl_pred', 'interactive_vis': False, 'show_largest_person_only': False, 'show_mesh_stand_on_image': False, 'soi_camera': 'far', 'make_tracking': False, 'temporal_optimization': False, 'save_dict_results': True, 'save_visualization_on_img': True, 'fps_save': 24, 'character': 'smpl', 'renderer': 'pytorch3d', 'f': None, 'model_return_loss': False, 'model_version': 1, 'multi_person': True, 'new_training': False, 'perspective_proj': False, 'FOV': 60, 'focal_length': 443.4, 'lr': 0.0003, 'adjust_lr_factor': 0.1, 'weight_decay': 1e-06, 'epoch': 120, 'fine_tune': True, 'GPUS': 0, 'batch_size': 64, 'input_size': 512, 'master_batch_size': -1, 'nw': 4, 'optimizer_type': 'Adam', 'pretrain': 'simplebaseline', 'fix_backbone_training_scratch': False, 'backbone': 'hrnet', 'model_precision': 'fp32', 'deconv_num': 0, 'head_block_num': 2, 'merge_smpl_camera_head': False, 'use_coordmaps': True, 'hrnet_pretrain': '/root/autodl-tmp/neuman/preprocess/ROMP/trained_models/pretrain_hrnet.pkl', 'resnet_pretrain': '/root/autodl-tmp/neuman/preprocess/ROMP/trained_models/pretrain_resnet.pkl', 'loss_thresh': 1000, 'max_supervise_num': -1, 'supervise_cam_params': False, 'match_preds_to_gts_for_supervision': False, 'matching_mode': 'all', 'supervise_global_rot': False, 'HMloss_type': 'MSE', 'eval': False, 'eval_datasets': 'pw3d', 'val_batch_size': 4, 'test_interval': 2000, 'fast_eval_iter': -1, 'top_n_error_vis': 6, 'eval_2dpose': False, 'calc_pck': False, 'PCK_thresh': 150, 'calc_PVE_error': False, 'centermap_size': 64, 'centermap_conf_thresh': 0.25, 'collision_aware_centermap': False, 'collision_factor': 0.2, 'center_def_kp': True, 'local_rank': 0, 'distributed_training': False, 'distillation_learning': False, 'teacher_model_path': '/export/home/suny/CenterMesh/trained_models/3dpw_88_57.8.pkl', 'print_freq': 50, 'model_path': 'trained_models/ROMP_HRNet32_V1.pkl', 'log_path': '/root/autodl-tmp/neuman/preprocess/log/', 'learn_2dpose': False, 'learn_AE': False, 'learn_kp2doffset': False, 'shuffle_crop_mode': False, 'shuffle_crop_ratio_3d': 0.9, 'shuffle_crop_ratio_2d': 0.1, 'Synthetic_occlusion_ratio': 0, 'color_jittering_ratio': 0.2, 'rotate_prob': 0.2, 'dataset_rootdir': '/root/autodl-tmp/neuman/preprocess/dataset/', 'dataset': 'h36m,mpii,coco,aich,up,ochuman,lsp,movi', 'voc_dir': '/root/autodl-tmp/neuman/preprocess/dataset/VOCdevkit/VOC2012/', 'max_person': 64, 'homogenize_pose_space': False, 'use_eft': True, 'smpl_mesh_root_align': False, 'Rot_type': '6D', 'rot_dim': 6, 'cam_dim': 3, 'beta_dim': 10, 'smpl_joint_num': 22, 'smpl_model_path': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/parameters', 'smpl_J_reg_h37m_path': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/parameters/J_regressor_h36m.npy', 'smpl_J_reg_extra_path': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/parameters/J_regressor_extra.npy', 'smpl_uvmap': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/parameters/smpl_vt_ft.npz', 'wardrobe': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/wardrobe', 'mesh_cloth': 'ghostwhite', 'nvxia_model_path': '/root/autodl-tmp/neuman/preprocess/ROMP/model_data/characters/nvxia', 'track_memory_usage': False, 'adjust_lr_epoch': [], 'kernel_sizes': [5], 'collect_subdirs': False, 'save_mesh': True, 'save_centermap': False}
INFO:root:------------------------------------------------------------------
visualize in gpu mode
INFO:root:start building model.
Using ROMP v1
Confidence: 0.25
INFO:root:using fine_tune model: trained_models/ROMP_HRNet32_V1.pkl
INFO:root:missing parameters of layers:['_result_parser.params_map_parser.smpl_model.betas', '_result_parser.params_map_parser.smpl_model.faces_tensor', '_result_parser.params_map_parser.smpl_model.v_template', '_result_parser.params_map_parser.smpl_model.shapedirs', '_result_parser.params_map_parser.smpl_model.J_regressor', '_result_parser.params_map_parser.smpl_model.J_regressor_extra9', '_result_parser.params_map_parser.smpl_model.J_regressor_h36m17', '_result_parser.params_map_parser.smpl_model.posedirs', '_result_parser.params_map_parser.smpl_model.parents', '_result_parser.params_map_parser.smpl_model.lbs_weights', '_result_parser.params_map_parser.smpl_model.vertex_joint_selector.extra_joints_idxs']
INFO:root:Train all layers, except: ['_result_parser.params_map_parser.smpl_model.betas']
visualize in gpu mode
Initialization finished!
Processing /root/autodl-tmp/data/my_video/output/images, saving to /root/autodl-tmp/data/my_video/output/smpl_pred
INFO:root:gathering datasets
Loading 23 images to process
Processed 0 / 23 images

Then it terminates. I really have no idea what to do. Could you please help me out of the situation?

**Comment by Arthur151** [comment]
@KevinzhouCUC 
Please try the simple romp
https://github.com/Arthur151/ROMP/tree/master/simple_romp
It is much easier to use and install with a clean implementation for inference only.
Best,
Yu
